packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -76,6 +90,43 @@ class HorizontalOverflow extends Foundation<HorizontalOverflowProps,  React.AllH
         this.setState({
             itemsHeight
         });
+
+        if (canUseDOM() && this.horizontalOverflowItemsRef.current) {
+            this.horizontalOverflowItemsRef.current.addEventListener("scroll", this.throttled);
+        }
+    }
+
+    /**
+     * React life-cycle method
+     */
+    public componentWillUnmount(): void {
+        if (canUseDOM() && this.horizontalOverflowItemsRef.current) {
+            this.horizontalOverflowItemsRef.current.removeEventListener("scroll", this.throttled);
+        }
+    }
+
+    /**
+     * Callback on scroll change
+     */
+    public onScrollChange = (): IScrollChange => {
+        let scrollObject: IScrollChange;
+        const isLtr: boolean = this.getLTR() === Direction.ltr;
+        const distanceRemaining: number =
+            this.horizontalOverflowItemsRef.current.scrollWidth - this.horizontalOverflowItemsRef.current.scrollLeft;
+
+        if (this.horizontalOverflowItemsRef.current.scrollLeft === 0) {
+            scrollObject = { start: isLtr, end: !isLtr };
+        } else if (distanceRemaining === this.horizontalOverflowItemsRef.current.clientWidth) {
+            scrollObject = { start: !isLtr, end: isLtr };
+        } else {
+            scrollObject = { start: false, end: false };
+        }
+
+        if (this.props.onScrollChange) {
+            this.props.onScrollChange(scrollObject);
+        }
+
+        return scrollObject;
```

Comment:
why does this return anything?

Reply:
The entire meetings conclusion said we would return an object with scroll data to the consumer.

Reply:
But that is what you are doing with `this.props.onScrollChange`

Reply:
The return value at the end of this does nothing.

Reply:
Got it